api/storage/app/lighthouse-schema.graphql

```diff
@@ -770,6 +770,15 @@ type Query {
   skillFamilies: [SkillFamily]!
   skill(id: ID!): Skill
   skills: [Skill]!
+  usersPaginated(
+    where: UserFilterInput
+
+    """Limits number of fetched items."""
+    first: Int!
```

Comment:
It is possible to set a default page size but I wasn't sure what we'd prefer so I'd left it.

I don't like the name `first` either but I can't seem to find a way of renaming it.

Reply:
OK, if it's not simple to change the name then let's leave it.  If you can set a default pages size let's do it.  I vote for 10. :laughing:

Reply:
> > Pagination without any sorting is a bit weird. There's really no guarantee from the database that you won't get back a different random set of items on your page for any given query if there's no sorting. We're not using a database cursor queue our rows into a page, right? I hate to blow scope but how hard would it be to add _something_ for sorting? Maybe default to sorting by the primary key so our results are at least deterministic?
> 
> yeah, it would be really handy to know how the results are ordered it might be an additional argument to pass in, not sure
> 
> rerunning the query doesn't seem to return a random order so some order is being followed

Yeah, I think it currently returns them in the order they are in the database. I'll change it though.